### PR TITLE
Deploy to Deis

### PR DESCRIPTION
**Pull request by glogiotatidis**
Travis configuration and scripts plus code changes needed to run on Deis.

### CODE REVIEW

File: snippets/settings.py [hunk 1]
```diff
@@ -185,3 +184,6 @@
 }
 
 GEO_URL = 'https://location.services.mozilla.com/v1/country?key=fff72d56-b040-4205-9a11-82feda9d83a3'  # noqa
+
+PROD_DETAILS_STORAGE = config('PROD_DETAILS_STORAGE',
+                              default='product_details.storage.PDFileStorage')
```
Comment (glogiotatidis) [inline snippets/settings.py]: The plan is to enable the db backend and at cron job to update it regularly.

**Comment by glogiotatidis** [comment]
@pmclanahan Please r?

File: snippets/settings.py [hunk 2]
```diff
@@ -44,6 +43,7 @@
     'smuggler',
     'django_filters',
     'django_ace',
+    'product_details',
```
Comment (pmclanahan) [inline snippets/settings.py]: I see django-redis installed but don't see any updates to settings for that. Is that missing or all just handled via the env?

Comment (pmclanahan) [inline snippets/settings.py]: Yeah, I guess you can just set `CACHE_URL` to a `redis://` URL and it'll work. Nice :)

**Comment by pmclanahan** [comment]
LGTM! r+ :snowman:

**Comment by glogiotatidis** [comment]
Thanks @pmclanahan !

**Pull request merged**